.yamato/csharp-tests.yml

```diff
@@ -0,0 +1,19 @@
+name: Mac Edit Mode Tests
+triggers:
+    branches:
```

Comment:
What about release and hotfix branches? What about merges into develop-* branches? (The latter is less common)

Reply:
Could update it to support those branches as well.  Adding multiple `triggers` sections `OR`s them together.

Reply:
Cool, shouldn't block this PR now, but probably something we'll want soon-ish (like whenever the next release is).

Reply:
fixed